packages/mdc-typography/README.md

```diff
@@ -2,22 +2,33 @@
 title: "Typography"
 layout: detail
 section: components
-excerpt: "Material Design typography guidelines implemented via CSS."
+excerpt: "Typographic scale that handles a set of type sizes"
 iconId: typography
 path: /catalog/typography/
 -->
 
 # Typography
 
-MDC typography is a CSS-only component that implements the
-[Material Design typography guidelines](https://material.io/guidelines/style/typography.html), and makes them available to
-developers as CSS classes.
+Material Design's typographic scale has a limited set of type sizes that work well together along with the layout grid.
+
+MDC Typography is a foundational module that applies that typographic scale to MDC Web components. The typographic styles in this module are derived from ten styles:
+
+* Display 4
+* Display 3
+* Display 2
+* Display 1
+* Headline
+* Title
+* Subheading
+* Body 2
+* Body 1
+* Caption
 
 ## Design & API Documentation
 
 <ul class="icon-list">
   <li class="icon-list-item icon-list-item--spec">
-    <a href="https://material.io/guidelines/style/typography.html">Material Design guidelines: Typography</a>
+    <a href="https://material.io/guidelines/style/typography.html">Material Design guidelines: Style – Typography</a>
```

Comment:
Is it standard for us to match the text in the top bar on those pages, as opposed to the first heading or the window title (which are both just Typography)?